src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -644,6 +652,27 @@ public virtual void SetDebuggerStepMode(bool enabled)
 
         #region Internal Methods
 
+        /// <summary>
+        /// Breaks into the debugger.
+        /// </summary>
+        /// <param name="triggerObject">The object that triggered the breakpoint, if there is one.</param>
+        internal virtual void Break(object triggerObject = null)
+        {
+            if (!IsDebugHandlerSubscribed &&
```

Comment:
This implementation code should go into the ScriptDebugger class below, since it applies only there.  This should throw "not implemented" exception.

Reply:
Fixed, thanks.